src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -0,0 +1,67 @@
+package mate.academy.spring.controller;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.model.dto.MovieSessionRequestDto;
+import mate.academy.spring.model.dto.MovieSessionResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.mapper.MovieSessionMapper;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movie-sessions")
+public class MovieSessionController {
+    private MovieSessionService movieSessionService;
+    private MovieSessionMapper movieSessionMapper;
+
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionMapper movieSessionMapper) {
+        this.movieSessionService = movieSessionService;
+        this.movieSessionMapper = movieSessionMapper;
+    }
+
+    @PostMapping
+    public MovieSessionResponseDto create(
+            @RequestBody MovieSessionRequestDto movieSessionRequestDto) {
+        return movieSessionMapper.toDto(movieSessionService
+                .add(movieSessionMapper.toModel(movieSessionRequestDto)));
+    }
+
+    @DeleteMapping("/{id}")
+    public void delete(@PathVariable Long id) {
+        movieSessionService.delete(id);
+    }
+
+    @PutMapping("/{id}")
+    public MovieSessionResponseDto update(
+            @RequestBody MovieSessionRequestDto movieSessionRequestDto,
+            @PathVariable Long id
+    ) {
+        MovieSession movieSession = movieSessionMapper.toModel(movieSessionRequestDto);
+        movieSession.setId(id);
+        movieSessionService.update(movieSession);
+        return movieSessionMapper.toDto(movieSession);
+    }
+
+    @GetMapping("/find")
```

Comment:
```suggestion
    @GetMapping("/available")
```

"find" - not very informative url...